website/content/maintainers/github/templates/required/governance-maintainer.md

```diff
@@ -1,11 +1,216 @@
 ---
 title: "The Maintainer Council Template"
 linkTitle: "Maintainer Governance"
-date: 2022-09-15
+date: 2022-12-02
 draft: false
 weight: 31
 ---
 
-These instructions are still being drafted.  In the meantime, view the
-[raw text of GOVERNANCE-maintainer.md](https://raw.githubusercontent.com/cncf/project-template/main/GOVERNANCE-maintainer.md),
- which contains instructions in markdown comments.
+| Audience of this HowTo | Audience of CONTRIBUTING.md | Required by CNCF      |
```

Comment:
Feh, should change the "template" for this header; that middle should be "Audience of the document" instead of requiring a change each time, because of this issue.   I'll change it on the gov docs.